Formula/easyengine.rb

```diff
@@ -0,0 +1,27 @@
+class Easyengine < Formula
+  desc "Command-line control panel to manage WordPress sites"
+  homepage "https://easyengine.io/"
+  url "https://github.com/EasyEngine/easyengine/releases/download/v4.0.0/easyengine.phar"
+  sha256 "5d7f7ec95911883240717458024f0e7c69309f7bd3646353a0594e1b0900eaa6"
+
+  bottle :unneeded
+
+  depends_on "php"
+
+  def install
+    bin.install "easyengine.phar" => "ee"
+  end
+
+  def caveats; <<~EOS
+    EasyEngine requires Docker to work.
+
+    You can install Docker using:
+
+      brew cask install docker
+  EOS
+  end
+
+  test do
+    system bin/"ee cli version"
```

Comment:
Since we have dependency on docker for all current feature set, other feature test case will not able to pass in CI.

We can add more test-cases in future when we have more command which don’t depend on docker. @SMillerDev

Reply:
I don't think this would be suitable for the core tap in that case especially since we can't get it to work without doing additional commands and not have it tested. You already have a viable way of installing it as shown on the website so I would stick to that for now imo.